usb/usb.go

```diff
@@ -166,6 +167,44 @@ func goBuild() error {
 	return nil
 }
 
+func wingo() error {
+	cmd := exec.Command("go", "generate", "github.com/u-root/wingo")
+	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
+	if err := cmd.Run(); err != nil {
+		return err
+	}
+	cmd = exec.Command("go", "install", "github.com/u-root/wingo")
+	cmd.Env = append(os.Environ(), "GOBIN="+filepath.Join(workingDir, "usr/bin"))
+	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
+	if err := cmd.Run(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func midori() error {
+	if err := os.MkdirAll("usr/bin", 0755); err != nil {
+		return err
+	}
+	resp, err := http.Get("https://nichromeos.org/index.php/s/3WDZWSb38HmdmKs/download")
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("non-200 HTTP status: %d", resp.StatusCode)
```

Comment:
It might be helpful to also log the body of the error but not a big deal. Kind of depends on what kind of message you might get back from `nichromeos.org`

Reply:
I used to do that in wget (where this code came from) but it turned out to be not real helpful, I backed out to just the error number. Sometimes you'd get back this giant load of html barf that was incomprehensible ... but thanks for a suggestion that *ought* to work well. Ah, web.